### PR TITLE
update doc for docker/kube features

### PR DESCRIPTION
**Pull request by xvello**
Update the docs reflecting the state of `beta9`.

### CODE REVIEW

File: docs/agent/changes.md [hunk 1]
```diff
@@ -212,57 +213,51 @@ file for an example configuration file.
...
 * Some options have moved from `docker_daemon.yaml` to the main `datadog.yaml`:
-  * `docker_root` option has been split in two options `container_cgroup_root`
-    and `container_proc_root`.
+  * `collect_labels_as_tags` has been renamed `docker_labels_as_tags` and now
+    supports high cardinality tags, see the details in `datadog.yaml.example`
   * `exclude` and `include` list have been renamed `ac_include` and
     `ac_exclude`. They will impact every container-related component of the
-    agent. Those only lists supports image name and container name (instead of
-    any tags).
+    agent. Those only lists supports `image` name and `container` name (instead of
```
Comment (masci) [inline docs/agent/changes.md]: Not sure this sentence is working well, can we rephrase?

File: docs/agent/changes.md [hunk 2]
```diff
@@ -212,57 +213,51 @@ file for an example configuration file.
...
 
-Much of the `utils` directory has been removed from the agent as well. However,
-most of what was removed was not diretly related to checks and wouldn't be imported
-in almost anyone's checks. The flare module, for example, was removed and
-reimplemented in Go, but is unlikely to have been used by anyone in a custom check.
-To learn more, you can read about the details in the [development documentation][python-dev].
+We are still working on kubernetes integration, feature parity for the `kubernetes`
+check will be provided by combining:
+  * The (work-in-progress) `kubelet` check retrieving metrics from the kubelet
+  * The (work-in-processs) `kubernetes_apiserver` check retrieving events and
```
Comment (masci) [inline docs/agent/changes.md]: s/work-in-process/work-in-progress

File: docs/beta.md [hunk 3]
```diff
@@ -30,16 +30,11 @@ ready yet. This is the list of checks that are expected to fail if run within th
 beta Agent:
 
 * agent_metrics
-* docker_daemon
-* kubernetes
-* kubernetes_state
+* docker_daemon [replaced by a new `docker` check](changes.md#docker-check)
+* kubernetes [to be replaced by new checks](changes.md#kubernetes-support)
```
Comment (masci) [inline docs/beta.md]: 🍰

**Review by masci — CHANGES_REQUESTED** [review]
Left a couple of comments, otherwise 💯

File: docs/agent/changes.md [hunk 4]
```diff
@@ -15,7 +15,8 @@ open an issue or submit a Pull Request.
 * [APM agent](#apm-agent)
 * [Process agent](#process-agent)
 * [Python Modules](#python-modules)
-* [Kubernetes support](#kubernetes-support)
+* [Docker]((#docker-check))
```
Comment (CharlyF) [inline docs/agent/changes.md]: Seems broken https://github.com/DataDog/datadog-agent/blob/4853eb4b2c99e15a3a44a95910f0acee0e5d41b9/docs/agent/(#docker-check)

File: docs/agent/changes.md [hunk 5]
```diff
@@ -15,7 +15,8 @@ open an issue or submit a Pull Request.
...
+* [Kubernetes](#kubernetes-support)
 * [JMX](#jmx)
 * [GUI](#gui)
```
Comment (CharlyF) [inline docs/agent/changes.md]: Worth listing the DCA ?

File: docs/agent/changes.md [hunk 6]
```diff
@@ -279,6 +275,22 @@ The following options and tags are deprecated:
 
 The `kube_service` tagging depends on the `Datadog Cluster Agent`, which is not released yet.
 
+## Python Modules
+
+While we are continuing to ship the python libraries that shipped with Agent 5,
```
Comment (CharlyF) [inline docs/agent/changes.md]: "the python libraries that shipped with Agent 5" > "the python libraries that ship with Agent 5" ?

File: docs/beta.md (hunk 3 above)
Comment (CharlyF) [inline docs/beta.md]: Seems like the two above links 404

**Review by CharlyF — CHANGES_REQUESTED** [review]
Thanks a bunch for this PR! Left a few comments.
There are 3 dead links, happy to approve when they are updated

File: docs/agent/changes.md [hunk 7]
```diff
@@ -279,6 +275,22 @@ The following options and tags are deprecated:
 
 The `kube_service` tagging depends on the `Datadog Cluster Agent`, which is not released yet.
```
[the remark this answers is not in the document]
Comment (xvello) [reply docs/agent/changes.md]: Once we officially release the DCA, we'll need a `docs/cluster-agent` folder and update that part and the kubernetes events section to link to it, indeed.

My first impression is that we should link from the dockerfiles readme to the doc, not the other way around, WDYT?

[the remark this answers is not in the document]
Comment (CharlyF) [reply docs/agent/changes.md]: yeah, that's fair 👍

**Review by CharlyF — APPROVED** [review]

**Pull request merged**